### PR TITLE
consensus/pbft: Add PBFT message logs

### PR DESCRIPTION
**Pull request by alanchchen**

### CODE REVIEW

File: consensus/pbft/message_set.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	messages *set.Set
+}
+
+type peerMessage struct {
+	peer    Peer
+	message interface{}
+}
+
+func (ms *messageSet) Sequence() *big.Int {
+	if ms == nil {
```
Comment (markya0616) [inline consensus/pbft/message_set.go]: ms == nil? it should be ms.view == nil?

File: consensus/pbft/message_set.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+		return new(big.Int)
+	}
+	return ms.view.Sequence
+}
+
+func (ms *messageSet) ViewNumber() *big.Int {
+	if ms == nil {
```
Comment (markya0616) [inline consensus/pbft/message_set.go]: ms == nil? it should be ms.view == nil?

File: consensus/pbft/message_set.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+		return big.NewInt(-1)
+	}
+	return ms.view.ViewNumber
+}
+
+func (ms *messageSet) Type() reflect.Type {
+	if ms == nil {
```
Comment (markya0616) [inline consensus/pbft/message_set.go]: should we check this?

Comment (alanchchen) [inline consensus/pbft/message_set.go]: Both are ok. We check `ms == nil` here to avoid checking it outside.

**Pull request merged**